server/game/cards/02_SHD/units/MigsMayfeldTriggerman.ts

```diff
@@ -0,0 +1,29 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+
+export default class MigsMayfeldTriggerman extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '6910883839',
+            internalName: 'migs-mayfeld#triggerman'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addTriggeredAbility({
+            title: 'You may deal 2 damage to a unit or base.',
+            limit: AbilityHelper.limit.perRound(1),
+            when: {
+                onCardsDiscardedFromHand: (event, context) => true,
+            },
+            targetResolver: {
+                optional: true,
+                immediateEffect: AbilityHelper.immediateEffects.damage(() => ({
+                    amount: 2,
```

Comment:
nit: since there's just one parameter it would be fine all in one line, but personal preference

Reply:
I come from the land of C# where separate lines are The Way. I'll move it to one line to meet js style norms.